src/ng-select/lib/ng-select.component.ts

```diff
@@ -787,12 +772,6 @@ export class NgSelectComponent implements OnDestroy, OnChanges, AfterViewInit, C
             });
     }
 
-    private _setInputValue() {
-      if (!this.multiple && this.selectedItems.length) {
-        this.searchTerm = this.selectedItems[0].label;
-      }
-    }
-
```

Comment:
@geebru any impact on functionality

Reply:
These got duplicated during the merge so this is just removing the duplicate code.